src/Components/CommonComponents/navigationBar.scss

```diff
@@ -0,0 +1,130 @@
+@import '../../styles.scss';
+
+.Navigation-bar {
+    @include flex-row();
+    position: relative;
+    justify-content: space-between;
+
+    .Navigation-menu {
+        @include flex-row();
+        gap: 20px;
+        padding: 20px 0 0 70px;
+
+        @media only screen and (max-width: 420px) {
+            gap: 0px;
+            padding: 10px 0 0 30px;
+        }
+
+
+        .Navigation-menu-logo {
+            width: 65%;
+            height: 65%;
+
+            @media (max-width:670px) {
+                width: 40%;
+                height: 40%;
+            }
+
+            @media only screen and (max-width: 420px) {
+                width: 70px;
+                height: 40px;
+            }
+        }
+
+        .Menu-btn {
+            width: 70px;
+            height: 50px;
+            background: none;
+            border: none;
+
+            @media (max-width:670px) {
+                width: 50px;
+                height: 40px;
+            }
+
+            @media only screen and (max-width: 420px) {
+                width: 50px;
+                height: 30px;
+            }
+        }
+
+        .Navigation-menu-nav-btn {
+            width: 100%;
+            height: 100%;
+
+            &:hover {
+                cursor: pointer;
+            }
+
+            &:active {
+                transition-duration: 0.05s;
+                box-shadow: 0 0 10px #ff9d72;
+            }
+        }
+    }
+
+    .Reservations-menu {
```

Comment:
why css is starting with capital letter?

Reply:
updated the classes starting with smallcase